client/signup/steps/intent/intent-screen.tsx

```diff
@@ -34,6 +34,19 @@ const useIntents = ( { translate }: Pick< Props, 'translate' > ): Intent[] => {
 			actionText: translate( 'Start building' ),
 		},
 	];
+
+	if ( isEnabled( 'seller-experience' ) ) {
+		intents.push( {
+			key: 'sell',
+			title: translate( 'Sell' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'sell', 'block search term' )` *ES Score: 10*